content/en/docs/concepts/architecture/nodes.md

```diff
@@ -9,10 +9,10 @@ weight: 10
 
 <!-- overview -->
 
-Kubernetes runs your workload by placing containers into Pods to run on _Nodes_.
+Kubernetes runs your workload by placing containers into pods to run on _nodes_.
```

Comment:
My reading of this sentence (as formatted) is that Node is being introduced as a concept.
Perhaps the italics is not needed?
When I read the word, nodes (lowercase), I think of a machine or VM.

Reply:
I don't know if we italicize introductory terms without also capitalizing, but to move forward let's leave this.

Reply:
> I don't know if we italicize introductory terms without also capitalizing, but to move forward let's leave this.

Sometimes we do.